### PR TITLE
NEO-875 Support selecting, dragging and resizing elements with touch inputs

### PR DESCRIPTION
**Pull request by jaller94**
Allows to select, move, resize and change the text of elements using touch.

Other than #574, this PR does not break the mouse input.

**:heavy_check_mark: Checklist**

- [ ] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Added or updated documentation.
- [ ] Tests for new functionality and regression tests for bug fixes.
- [ ] Screenshots or videos attached (for UI changes).
- [ ] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

**Review by weeman1337 — APPROVED** [review]
Tested on desktop with Firefox and Chromium. Works :heavy_check_mark: 

Would be great if someone can review it with a real touch device before merge.

Good work @jaller94 !

File: packages/react-sdk/src/components/Whiteboard/ElementBehaviors/Selection/DragSelect.test.tsx [hunk 1]
```diff
@@ -105,38 +105,38 @@ describe('<DragSelect/>', () => {
...
+    fireEvent.pointerMove(screen.getByTestId('drag-select-layer'), {
       clientX: 50,
       clientY: 50,
     });
 
     expect(screen.queryByTestId('drag-selection')).not.toBeInTheDocument();
   });
 
-  it('should render a selection if there is a mouse move and start coordinates', () => {
+  it('should render a selection if there is a ponter move and start coordinates', () => {
```
Comment (mgcm) [inline packages/react-sdk/src/components/Whiteboard/ElementBehaviors/Selection/DragSelect.test.tsx:131]: pointer:)

**Review by mgcm — CHANGES_REQUESTED** [review]

**Comment by mgcm** [comment]
Trying this with an actual touch device (iPad 6th gen with iOS 17.7) and I was able to:

- create a new shape by tapping the tool and then the whiteboard
- resize and move a selected shape after creating it by tapping the board (very slow with lagging behind touch point)
- selecting other shapes only works when a shape is already selected (??) and it's very slow (takes one second between tapping and the shape selection becomes active)
- selecting shapes stops working if you delete the selected shape, for example

What didn't work:

- could not create shapes by dragging
- can't change the text, keyboard input is never activated (what is the supposed interaction here? double tap to edit text?)
- can't multi-select by dragging (dragging moves the whole whiteboard)
- can't use other dragging tools like arrows, lines and pencil
- there is no way to activate context-menus (secondary mouse click equivalent)

File: packages/react-sdk/src/components/Whiteboard/ElementBehaviors/Selection/DragSelect.test.tsx (hunk 1 above)
Comment (jaller94) [reply packages/react-sdk/src/components/Whiteboard/ElementBehaviors/Selection/DragSelect.test.tsx:131]:
```suggestion
  it('should render a selection if there is a pointer move and start coordinates', () => {
```

**Review by mgcm — APPROVED** [review]

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**